packages/main/src/plugin/kubernetes/kubernetes-port-forward-service.ts

```diff
@@ -73,7 +75,10 @@ export class KubernetesPortForwardService implements IDisposable {
   constructor(
     private configManagementService: ConfigManagementService,
     private forwardingConnectionService: PortForwardConnectionService,
-  ) {}
+    private apiSender: ApiSenderType,
+  ) {
+    this.apiSender.send('kubernetes-port-forwards-update');
```

Comment:
Not needed in the constructor ?

Reply:
I added it there in order to clear the store when there is a context switch (thus a new service is created)

Reply:
Noted!